public-api/src/main/java/org/pmiops/workbench/publicapi/DataBrowserController.java

```diff
@@ -262,14 +262,19 @@ public ConceptAnalysis apply(ConceptAnalysis ca) {
             standardConceptFilter = StandardConceptFilter.STANDARD_OR_CODE_ID_MATCH;
         }
 
+        List<String> domainIds = null;
+        if (searchConceptsRequest.getDomain() != null) {
+            domainIds = DOMAIN_MAP.get(searchConceptsRequest.getDomain()).asList();
+        }
+
         if(searchConceptsRequest.getQuery() == null || searchConceptsRequest.getQuery().isEmpty()){
```

Comment:
One thing that may simply the logic and code here is just "If there is no query and the standard concept filter = STANDARD_OR_CODE_ID_MATCH   then set standard concept filter = STANDARD_CONCEPTS ... and the code continue.  or put an or below to catch the standard or code ...

Reply:
Done.